kubeprod/cmd/install.go

```diff
@@ -33,8 +33,9 @@ import (
 const (
 	FlagManifests          = "manifests"
 	defaultManifestBaseFmt = "https://github.com/bitnami/kube-prod-runtime/raw/%s/manifests/"
-
-	FlagPlatformConfig = "config"
+	FlagOnlyGenerate       = "only-generate"
+	FlagOnlyGenerateShort  = "g"
```

Comment:
I wouldn't add a short option for this - it seems too complicated an option to summarise as "`-g`".  Even more so when we don't offer anything else via a short option.